modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNode.tsx

```diff
@@ -269,7 +269,7 @@ export function ObjectDefinitionNode({
 					{Object.keys(nodeHandleRefs).map((position, index) => (
 						<Handle
 							className="lfr-objects__model-builder-node-handle"
-							id={id.toString()}
+							id={index.toString()}
```

Comment:
Using the index here will cause the problem that all nodes in the flow will have the same handle ids: 1, 2 3 and 4. This won't allow that the user makes a line from the same node to another, like the bottom handle from one node to the bottom handle of another node (see video above). This happens because of the previous condition you added on the `Diagram.tsx`, so my suggestion is for you to use both the id and the index: 

``id={`${id}_${index}`}``

Reply:
Great observation @rebsilva! I hadn't realized that this would fall into the new validation.

Reply:
Hey @rebsilva, To make this code more coherent, I was thinking about changing the index to position. What do you think? Then we would have `8293_Left`, `8293_Right`....

Reply:
I think its a good idea @igor-franca!